app/src/main/java/com/yeonae/chamelezone/view/place/PlaceReviewTabFragment.kt

```diff
@@ -29,31 +29,39 @@ import kotlinx.android.synthetic.main.item_place_review.*
 
 class PlaceReviewTabFragment : Fragment(), PlaceReviewContract.View {
     override lateinit var presenter: PlaceReviewContract.Presenter
+    private lateinit var placeReviewRvAdapter: PlaceReviewTabRvAdapter
     private var reviewNumber = 0
     private var placeNumber = 0
     private var memberNumber = 0
-    var placeName = arguments?.getString(PLACE_NAME)
+    var reviewMemberNumber = 0
+    var placeName: String = ""
 
-    private lateinit var placeReviewRvAdapter: PlaceReviewTabRvAdapter
 
-    override fun showPlaceReview(reviewList: List<ReviewResponse>) {
+    override fun showPlaceReview(reviewList: List<ReviewItem>) {
         if (::placeReviewRvAdapter.isInitialized)
             placeReviewRvAdapter.addData(reviewList)
     }
 
     override fun showReviewDelete(message: String) {
-//        ll_review.isGone = true
         Toast.makeText(context, "리뷰가 삭제되었습니다", Toast.LENGTH_LONG).show()
     }
 
     override fun showMemberReview(user: UserEntity) {
-
+        reviewMemberNumber = placeReviewRvAdapter.reviewMemberNumber
+        Log.d("PlaceReviewTabFragment memberNumber", memberNumber.toString())
+        Log.d("PlaceReviewTabFragment reviewMemberNumber2", reviewMemberNumber.toString())
+        if (memberNumber == reviewMemberNumber) {
+            btn_more.isVisible = true
+        } else if (memberNumber == 0) {
+            btn_more.isInvisible = true
+        }
     }
 
     override fun getMemberCheck(response: Boolean) {
         presenter.getMember()
         if (response) {
             review.setOnClickListener {
+                placeName = arguments?.getString(PLACE_NAME).toString()
```

Comment:
```suggestion
                placeName = arguments?.getString(PLACE_NAME).orEmpty()
```
먼저 작업하신 케이스에서는, 앞의 실행결과가 null일경우에 바로 toString을 하게되면 ,
그대로 "null"이라는 문자열 4글자가 반환됩니다.
원하는 동작이 아니겠죠?

Reply:
수정했습니다
c99a5cc